### PR TITLE
Stub value set creation + #2488

### PR DESCRIPTION
**Pull request by sheridancook**
created stub vs for practitionerspecialty, qualifiedroletype, healthcareproviderroletype based on request + QA clean up in VS

Fixed older parameter codes - special-uri -> special url

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,4 @@ input-cache/txcache/current/.capabilityStatement.cache
 input-cache/txcache/all-systems.cache
 input-cache/txcache/iso3166-2.cache
 /input-cache
+/input-cache
```
Comment (ElliotSilver) [inline .gitignore:32]: Please review this file:
* I see 5 repetitions of _gencontinuous.bat, etc.
* I've never seen `*.xsd` in an IG .gitignore.
* there shouldn't be a need for specific files in `input-cache` if we include `input-cache`, although I usually would write `/template/**`

**Review by ElliotSilver — COMMENTED** [review]
please review and update .gitignore

File: .gitignore (hunk 1 above)
Comment (sheridancook) [reply .gitignore]: @ElliotSilver  - Changes applied to .ignore file

**Review by ElliotSilver — APPROVED** [review]
++good.

**Pull request merged**